client/src/components/ReviewStarsByDescriptor.jsx

```diff
@@ -1,15 +1,46 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
+import StarsByDescriptor from './StarsByDescriptor';
+
+import {
+  Wrapper,
+  ColumnWrapper,
+  DescriptorWrapper,
+  DescriptorText,
+} from './styles/ReviewStarsByDescriptor.styles';
+
 const ReviewStarsByDescriptor = props => (
-  <div>
-    <h6> Accuracy {props.accuracyAvg} </h6>
-    <h6> Communication {props.communicationAvg} </h6>
-    <h6> Cleanliness {props.cleanlinessAvg} </h6>
-    <h6> Location {props.locationAvg} </h6>
-    <h6> Check-in {props.checkinAvg} </h6>
-    <h6> Value {props.valueAvg} </h6>
-  </div>
+  <Wrapper>
+    <ColumnWrapper>
+      <DescriptorWrapper>
+        <DescriptorText> Accuracy </DescriptorText>
+        <StarsByDescriptor avg={props.accuracyAvg} />
+      </DescriptorWrapper>
+      <DescriptorWrapper>
+        <DescriptorText> Communication </DescriptorText>
+        <StarsByDescriptor avg={props.communicationAvg} />
+      </DescriptorWrapper>
+      <DescriptorWrapper>
+        <DescriptorText> Cleanliness </DescriptorText>
+        <StarsByDescriptor avg={props.cleanlinessAvg} />
+      </DescriptorWrapper>
+    </ColumnWrapper>
+    <ColumnWrapper>
```

Comment:
Also is there a reason that there are column wrappers here? If it's to split the stars up by column I recommend flex-wrap. You can find the height of the columns that you want (CSS min-height), and if you set flex-wrap, it will wrap accordingly.

Reply:
I am both happy and and nervous that you pointed this out ha.  It took my a long time to dial in the styling for that section and I played around with the column wrappers specifically a lot! 
 Part of the tricky part was getting the space between the descriptor and stars to spread all the way out after you shrink the browser width down to the point where all six descriptors stack on one another.  BUT.  All that said I think you are probably right and it's worth revisiting.  This might be a "if I have time" refactor though!